### PR TITLE
Edgescan adding vulnerability annotation

### PR DESCRIPTION
**Pull request by mdrobniu**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This change is introducing a new command for adding annotations into vulnerabilities

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
@mdrobniu Is this ready for review?

File: Packs/Edgescan/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Edgescan
+- New command for adding annotations into vulnerabilities
```
Comment (tomneeman151293) [inline Packs/Edgescan/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Added the **edgescan-vulnerabilities-add-annotation** command for adding annotations into vulnerabilities.
```

**Comment by mdrobniu** [comment]
@tomneeman151293 Yes please.

**Review by tomneeman151293 — CHANGES_REQUESTED** [review]
Looks great! small change requested in RN to follow our RN conventions if possible.
Also, could you please add the command example to the README file of the integration?

**Comment by mdrobniu** [comment]
@tomneeman151293 Hey. I'm back from holiday. 

Can you please reenable write permissions for the repo? I know I was denied after PR to this repo and I'm unable now to push the changes here? 

Thanks

**Comment by tomneeman151293** [comment]
@mdrobniu Hey, I hope you have enjoyed your vacation. Unfortunately, due to a change in the policy of repository permissions, I can't give you permissions to that. Could you please open the PR via contribution flow.

**Comment by mdrobniu** [comment]
@tomneeman151293 
Ok I'll do. Please close this PR.

**Pull request closed without merging**